docs/reference/release-notes.asciidoc

```diff
@@ -17,7 +16,6 @@ This section summarizes the changes in each release.
 --
 
 include::release-notes/8.1.asciidoc[]
-include::release-notes/8.0.0.asciidoc[]
```

Comment:
@pugnascotia this was done automatically by `./gradlew generateReleaseNotes`, I do not think this is intended